webant/templates/edit-volume.html

```diff
@@ -0,0 +1,302 @@
+            <div class="form-group custom-field-form">
+              <div class='col-sm-12'>
+                <div class="custom-key">
+                    <input class="form-control" value="{{ p_key }}" type="text">
+                </div>
+                <div class='custom-value'>
+                    <input class="form-control" value="{{ p_value}}" type="text">
+                </div>
+                <div class='remove-custom-field'>
+                   <button type="button" class="close" aria-label="remove custom field">
+                    <span aria-hidden="true" class="glyphicon glyphicon-remove-circle"></span>
+                   </button>
+                </div>
+             </div>
+            </div>
+            {% endif %}
+        {% endfor %}
+        </div>
+
+        <div id='templates' hidden>
+            <div class="form-group custom-field-form">
+              <div class='col-sm-12'>
+                <div class="custom-key">
+                    <input class="form-control" placeholder="{%trans%}custom key{%endtrans%}" type="text">
+                </div>
+                <div class='custom-value'>
+                    <input class="form-control" placeholder="{%trans%}custom value{%endtrans%}" type="text">
+                </div>
+                <div class='remove-custom-field'>
+                   <button type="button" class="close" aria-label="remove custom field">
+                    <span aria-hidden="true" class="glyphicon glyphicon-remove-circle"></span>
+                   </button>
+                </div>
+             </div>
+            </div>
+        </div>
+
+        <div class="form-group" style="margin-top:-10px">
+            <div class="col-xs-12" align="right">
+                <span id="custom-fields-plus-button" onClick="addCustomField(true)" style="margin-right: 10px" data-toggle="tooltip" data-trigger="hover" data-placement="auto left" title="add another custom field" class="glyphicon glyphicon glyphicon-plus"></span>
+            </div>
+        </div>
+
+        <br>
+
+        <div class="form-group" align="center">
+            <button id="submit-button" class="btn btn-primary" type="submit" >{%trans%}Update{%endtrans%}</button>
+            <button id="updating-button" class="btn btn-primary" style='display: none' disabled="disabled"><span class='glyphicon glyphicon-refresh spin'></span></button>
+        </div>
+
+    </form>
+
+
+    <div id='success-modal' class="modal" tabindex="-1" role="dialog">
+      <div class="modal-dialog">
+        <div class="modal-content">
+          <div class="modal-body text-center">
+            <p style='font-size:17px'><span class='glyphicon glyphicon-ok' style='color: green'></span> {%trans%}The volume has been successfully updated{%endtrans%}</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">{%trans%}Cuntinue editing{%endtrans%}</button>
```

Comment:
On 02/29/2016 12:07 AM, particella wrote:
> In webant/templates/edit-volume.html
> <https://github.com/insomnia-lab/libreant/pull/250#discussion_r54358589>:
> 
>> +        <div class="form-group" align="center">
>> +            <button id="submit-button" class="btn btn-primary" type="submit" >{%trans%}Update{%endtrans%}</button>
>> +            <button id="updating-button" class="btn btn-primary" style='display: none' disabled="disabled"><span class='glyphicon glyphicon-refresh spin'></span></button>
>> +        </div>
>> +
>> +    </form>
>> +
>> +
>> +    <div id='success-modal' class="modal" tabindex="-1" role="dialog">
>> +      <div class="modal-dialog">
>> +        <div class="modal-content">
>> +          <div class="modal-body text-center">
>> +            <p style='font-size:17px'><span class='glyphicon glyphicon-ok' style='color: green'></span> {%trans%}The volume has been successfully updated{%endtrans%}</p>
>> +          </div>
>> +          <div class="modal-footer">
>> +            <button type="button" class="btn btn-default" data-dismiss="modal">{%trans%}Cuntinue editing{%endtrans%}</button>
> 
> You mean 'Continue editing' ('o' instead of 'u') :)

Yeap, that was a typo. I've fixed that.